packages/mui-material/src/Modal/Modal.js

```diff
@@ -237,23 +226,6 @@ Modal.propTypes /* remove-proptypes */ = {
    * @default false
    */
   keepMounted: PropTypes.bool,
-  /**
```

Comment:
This is suspicious, why were these removed? Should we update the props interface for the @mui/material's Modal?

Reply:
These were removed in an older commit. After #32739 was merged, these lines were restored.